SAMPLES.md

```diff
@@ -0,0 +1,147 @@
+In this page you can find some samples of generated code using swagger-gradle-codegen for every platform:
+
+## `kotlin` platform
+
+### Data Class
+
+The following definition:
+
+```json
+"definitions": {
+        "Pet": {
```

Comment:
Please keep indentation uniform (now you have 8 spaces between `definitions` and `Pet` and 4 spaces everywhere-else)